packages/vue/src/Form/ComplexCalendar/OcComplexCalendar.vue

```diff
@@ -9,10 +9,6 @@ defineProps({
   shortcuts: Array,
   cancelButtonProps: Object,
   submitButtonProps: Object,
-  /**
-   * Props to pass to the calendar component from https://vcalendar.io/
-   */
-  calendarProps: Object,
```

Comment:
is it safe to remove these props ?

Reply:
Yeah. I checked on the VCalendar.
can not find any props with same name
- https://github.com/nathanreyes/v-calendar/blob/master/src/components/Calendar.vue#L272-L294
- Also Playground with `OcComplexDatePicker` & `OcComplexCalendar`

Reply:
cc @mykhailo-diakovych